patch.js

```diff
@@ -1,9 +1,46 @@
 const fs = require('node:fs');
 const crypto = require('node:crypto');
 const path = require('node:path');
-const { pki, md } = require('node-forge');
+const { asn1, pki, md } = require('node-forge');
 const prompt = require('prompt');
 const colors = require('@colors/colors/safe');
+const dotenv = require('dotenv');
+
+const defaultOptions = {
+	nintendo_ca_g3_path: './CACERT_NINTENDO_CA_G3.der',
+	nintendo_ca_g3_format: 'der',
+	ca_private_key_path: undefined,
+	site_private_key_path: undefined,
+	csr_path: undefined,
+	common_name: '*',
+	output_folder_path: './'
+};
+
+async function main() {
+	dotenv.config();
+
+	if (process.argv.includes('-i') || process.argv.includes('--interactive')) {
+		showPrompt();
+
+		return;
+	}
+
+	const options = {
+		nintendo_ca_g3_path: process.env.NINTENDO_CA_G3_PATH || defaultOptions.nintendo_ca_g3_path,
+		nintendo_ca_g3_format: process.env.NINTENDO_CA_G3_FORMAT || defaultOptions.nintendo_ca_g3_format,
+		ca_private_key_path: process.env.CA_PRIVATE_KEY_PATH || defaultOptions.ca_private_key_path,
+		site_private_key_path: process.env.SITE_PRIVATE_KEY_PATH || defaultOptions.site_private_key_path,
+		csr_path: process.env.CSR_PATH || defaultOptions.csr_path,
+		common_name: process.env.COMMON_NAME || defaultOptions.common_name,
+		output_folder_path: process.env.OUTPUT_FOLDER_PATH || defaultOptions.output_folder_path
+	};
```

Comment:
Using multiple letters in a short argument is non-standard, but I can work with this.

Reply:
Sure, but there is no one set standard and you'd be hard pressed to find any software that conforms to them all. It depends on which standards you're looking at. If you go by [GNU standards](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) for example then software like ffmpeg is "non standard", since these standards state

> Option names are single alphanumeric characters

And ffmpeg has many arguments which violate this, such as `-b:v` for setting the bitrate